### DISCUSSION TITLE
Flutter sign-in key

### DISCUSSION THREAD

**Discussion by Sowjanya2527** [community/community#124068] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I tried to use the Flutter sign-in key to produce an apk/bundle, but even though I entered the correct path in the key.properties file, it still displays an error about a saved key path. Are you able to assist me, guys?

**Comment by Chittu13** [comment]
In the key.properties file, try to remove the single quotation or double inverted comma while providing the location.
Or, make sure the key store file's name is identical and appears twice in the same directory, or something similar.

**Comment by ebndev** [comment]
Hi @Sowjanya2527 & @Chittu13, thanks for being a part of the GitHub Community!

Please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
